### ISSUE TITLE
strange commit message

### ISSUE THREAD

**Issue by alok** [jreybert/vimagit#107]
I'm using latest version of `next` and I hit `CC` and it prefills the commit message `Bulk Update`. None of my commit messages for that repo have that message. I'm really confused here as to why that's being added. Is it a vimagit feature that I'm unaware of?

**Comment by alok** [comment]
It adds that line over and over every time I do an action like refreshing the buffer.  [Here's a gist of profiling that shows which functions are called.](https://gist.github.com/54e28101ec5e483ae787ecde09b98c01)

**Comment by jreybert** [comment]
I don't know from were this message could be from...

Could you please:
* `grep -R` your `.vim` directory to check if this message is a side effect of your configuration
* Precise what is your `vim` (or `neovim`) version and your `git` version

**Comment by alok** [comment]
`git version 2.7.0.25.gfc10eb5`
`NVIM v0.2.0-104-g5f02608`

Nothing comes up with `grep`, even under case-insensitive search.

**Comment by jreybert** [comment]
Did you try with `vim -U none`, and load only vimagit to check if there a side effect from another plugin or a setting in your vimrc?

If you can determinate that only vimagit is in fault, could you please try to dump some debug traces?
in vimagit buffer, type this:

``` viml
:set verbosefile=/tmp/vimagit.log
:XXverbose call magit#update_buffer()
```

Try with XX from 1 to 20, and stop when you encounter `Bulk Update` in verbose file.

**Comment by jreybert** [comment]
@Alok do you still encounter this problem?

**Comment by alok** [comment]
Sometimes, but I'm never able to reproduce it consistently, so I've basically given up on it.

**Comment by jreybert** [comment]
Next time you encouter it, please try this:
```
:set verbosefile=/tmp/vimagit.log
:XXverbose call magit#update_buffer()
```

**Comment by jreybert** [comment]
One more thing: when you say you can't reproduce it, do you mean that if it appears a first time, if you go out commit and retry `CC`, it does not appear again?

**Comment by ramok** [comment]
Step for reproduce:
1. Stage some hunk
2. CC start writing commit message
3. Press '?' in command mode to show help.
Help will rewrite commit message
4. Press '?' again. In commit message I get left
>
>        amend) with message written in this section
> ==============
5. Pressing '?' again and again add two lines every time
>
> ==============

**Comment by jreybert** [comment]
@ramok, the bug you are describing looks like #110 . It has been fixed in last release 1.7.0. Which vimagit version are you using? (SHA1)

**Comment by ramok** [comment]
My bad. It's was old verision. After update everything fine. Thanks
